armi/reactor/converters/axialExpansion/axialExpansionChanger.py

```diff
@@ -0,0 +1,396 @@
+# Copyright 2023 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Enable component-wise axial expansion for assemblies and/or a reactor."""
+
+from armi import runLog
+from armi.reactor.flags import Flags
+from armi.reactor.converters.axialExpansion import (
+    getSolidComponents,
+    _getDefaultReferenceAssem,
+)
+from armi.reactor.converters.axialExpansion.expansionData import ExpansionData
+from armi.reactor.converters.axialExpansion.assemblyAxialLinkage import (
+    AssemblyAxialLinkage,
+)
+from numpy import array
```

Comment:
```suggestion
from numpy import array

from armi import runLog
from armi.reactor.flags import Flags
from armi.reactor.converters.axialExpansion import (
    getSolidComponents,
    _getDefaultReferenceAssem,
)
from armi.reactor.converters.axialExpansion.assemblyAxialLinkage import (
    AssemblyAxialLinkage,
)
from armi.reactor.converters.axialExpansion.expansionData import ExpansionData
```

Reply:
Nothing exciting.

1. I moved the `numpy` import before the `armi` import.
2. I alphabetized your `armi` imports.